cli/options.go

```diff
@@ -290,6 +269,24 @@ func getConfigPathsFromOptions(options *ProjectOptions) ([]string, error) {
 	}
 }
 
+func absolutePaths(p []string, pwd string) ([]string, error) {
+	var paths []string
+	for _, f := range p {
+		if f == "-" {
+			paths = append(paths, f)
+			continue
+		}
+		if !filepath.IsAbs(f) {
+			f = filepath.Join(pwd, f)
+		}
+		if _, err := os.Stat(f); err != nil {
+			return paths, err
+		}
+		paths = append(paths, f)
```

Comment:
Why not transforming this into an absolute path anymore here ?

Reply:
because we just did on line 279
I'm assuming here we pass an absolute `pwd`, which should be either os.Gwd or project.WorkingDir

Reply:
ok, as the PR for absolute paths on working dir has been merged